src/controller/CommissioningDelegate.h

```diff
@@ -683,12 +683,20 @@ struct GeneralCommissioningInfo
 // the ICD Management cluster, and is used to communicate that information.
 struct ICDManagementClusterInfo
 {
+    bool isICD = false;
```

Comment:
I'm not sure i understand the added benefit of an isICD flag. Is this to be able to check after commissioning if the device is an icd?

Reply:
removed, thanks